### ISSUE TITLE
Thread title is always empty

### ISSUE THREAD

**Issue by marcus-gerhardy** [sulu/SuluCommentBundle#54]
| Q | A
| --- | ---
| Bug? | Yes
| New Feature? | no
| Comment Bundle Version | 2.0.0
| Browser Version | Chrome Version 124.0.6367.118 (Official Build) (64-bit)

#### Actual Behavior

After creating a comment, the thread title is empty

#### Expected Behavior

Thread title should be filled in with what's passed as a template variable as described in the official docs

#### Steps to Reproduce

1. Include the comment section in your page template (see docs):
```
{{ render(path('sulu_comment.get_threads_comments', {
    threadId: 'page-' ~ uuid, 
    referrer: app.request.uri,
    threadTitle: 'This is my thread title', 
    _format: 'html'
})) }}
```
2. Fill in a comment
3. Send comment
4. Thread title in admin area is empty

**Comment by Prokyonn** [comment]
Hey, thank you for reporting.
This issue should already be fixed with https://github.com/sulu/SuluCommentBundle/pull/51/files

Can you try out the 2.x branch and check if it works correctly?

**Comment by marcus-gerhardy** [comment]
Hey, i checked out the 2.x branch and the problem is solved. Thank you.

**Comment by alexander-schranz** [comment]
fixed in #51 released as 2.0.1
